migrations/2_deploy_root_chain_contracts.js

```diff
@@ -37,33 +41,45 @@ const libDeps = [
   }
 ]
 
-module.exports = async(deployer) => {
+module.exports = async(deployer, network, accounts) => {
   console.log('linking libs...')
   await bluebird.map(libDeps, async e => {
     await deployer.deploy(e.lib)
     deployer.link(e.lib, e.contracts)
   })
 
   console.log('deploying contracts...')
-  await deployer.deploy(RootChainManager)
-  await deployer.deploy(RootChainManagerProxy, RootChainManager.address)
+  const rootChainManager = await deployer.deploy(RootChainManager)
+  const rootChainManagerProxy = await deployer.deploy(RootChainManagerProxy, '0x0000000000000000000000000000000000000000')
+  await rootChainManagerProxy.updateAndCall(RootChainManager.address, rootChainManager.contract.methods.initialize(accounts[0]).encodeABI())
```

Comment:
Can we remove `_proxyTo` from `UpgradableProxy` constructor since we are always passing it as zero and actually setting it only using `updateAndCall` function?

Reply:
yes we can remove that